apps/api-gateway/src/user/user.controller.ts

```diff
@@ -222,37 +122,58 @@ export class UserController {
 
   }
 
-  @Get('public-profile')
+  @Get('/activity')
   @ApiOperation({
-    summary: 'Fetch user details',
-    description: 'Fetch user details'
-  })
-  @ApiQuery({
-    name: 'id',
-    type: Number,
-    required: false
+    summary: 'organization invitations',
+    description: 'Fetch organization invitations'
   })
-  async getPublicProfile(@User() reqUser: user, @Query('id') id: number, @Res() res: Response): Promise<object> {
-    const userData = await this.userService.getPublicProfile(id);
+  @UseGuards(AuthGuard('jwt'))
+  @ApiBearerAuth()
+  @ApiQuery({ name: 'limit', required: true })
+  @Roles(OrgRoles.OWNER, OrgRoles.SUPER_ADMIN, OrgRoles.ADMIN, OrgRoles.ISSUER, OrgRoles.VERIFIER, OrgRoles.MEMBER)
```

Comment:
Please remove the `Roles` and `OrgRoleGuard` from the user activity API. It is user specific API